smarts/env/gymnasium/wrappers/metrics.py

```diff
@@ -120,16 +110,16 @@ def step(self, action: Dict[str, Any]):
             # Update stored costs.
             self._records[self._scen_name][agent_name].record.costs = costs
 
-            if dones[agent_name]:
+            if info[agent_name]["done"]:
                 self._done_agents.add(agent_name)
                 if not (
-                    agent_obs.events.reached_goal
-                    or len(agent_obs.events.collisions) > 0
-                    or agent_obs.events.off_road
-                    or agent_obs.events.reached_max_episode_steps
+                    agent_obs["events"]["reached_goal"]
+                    or agent_obs["events"]["collisions"]
```

Comment:
What will be the output of `agent_obs["events"]["collisions"]` ?

Reply:
It would be a boolean. Truthfully this should be called "collided", or be a list of agent ids.